src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/PackTaskLogic.cs

```diff
@@ -575,5 +582,22 @@ private void PopulateProjectAndPackageReferences(PackageBuilder packageBuilder,
                 }
             }
         }
+
+        private static IDictionary<string, string> ParsePropertiesAsDictionary(string[] properties)
+        {
+            var dictionary = new Dictionary<string, string>();
+            foreach (var item in properties)
+            {
+                string[] pair = item.Split('=');
+                if (pair.Length != 2)
```

Comment:
What about `desc="In my description I ought to use = and ; characters just for fun"`?
Is this supported?